### DISCUSSION TITLE
Alternatives to mirroring renovate runner

### DISCUSSION THREAD

**Discussion by aishwarya257** [renovatebot/renovate#10279] [Request Help]
Is there a possibility to sync with version updates from https://gitlab.com/renovate-bot/renovate-runner without mirroring. 

I have some additional configurations in renovate config in the mirrored repository and because of that mirroring fails.

We had an idea to create a scheduled pipeline to create a scheduled pipeline to merge upstream changes and push to our repo. But we wanted to know if there is a better or alternative way to do it?

P.S: I also tried to add the mirrored repository in the autodiscoverFilter in config. But that did not work as well.

**Comment by viceice** [comment]
I don't think so.

Solution:
* Use mirror for renovate-runner to own gitlab instance.
* Create a second repo where you referring to your mirror. So you can easily add more customizations.
